src/services/validation.js

```diff
@@ -1,17 +1,15 @@
 import firebaseAPI from "@/services/API/checkUser.js";
 
-/* this function validates the entry against the database (for this usecase, it is firestore). 
-firebaseAPI service is used to validate and it returns a boolean value, indicating whether the user is valid or not.
-If the user is valid and validateCount = 0, then the function returns.
-If the user is not valid and validateCount = 0, then the user is not valid, so validateCount becomes 1 and login message displays an error, allowing the user to correct their entry.
-If the user is not valid and validateCount = 1, then the user has already been authenticated once before and the user has retyped, yet the entry is still invalid, then validateCount becomes 2 to indicate this behaviour.
-@params {String} - userID - the ID that needs to be validated 
-@params {Number} - validateCount - indicates how many times the user has been validated 
-@params {Boolean} - isCurrentUserValid - holds the value returned by the API */
-export async function validateSRN(userID, validateCount, isCurrentUserValid){
+/** This function validates an entry against the database (for this usecase, it is firestore). 
+* firebaseAPI service is used to validate and it returns a boolean value, indicating whether the user is valid or not.
+* @param {String} userID - current ID being authenticated 
+* @param {Number} validateCount - indicates how many times the user has been validated 
+*/
+
+export async function validateSRN(userID, validateCount){
     var invalidLoginMessage = ""
 
-    isCurrentUserValid = await firebaseAPI.checkUserExists(userID);
+    var isCurrentUserValid = await firebaseAPI.checkUserExists(userID);
```

Comment:
```suggestion
    let isCurrentUserValid = await firebaseAPI.checkUserExists(userID);
```
We don't need this variable to be scoped outside of this method, so no need of `var`